packages/mobile/src/screens/edit-track-screen/screens/ListSelectionScreen.tsx

```diff
@@ -44,6 +45,7 @@ const useStyles = makeStyles(({ spacing, typography }) => ({
     justifyContent: 'space-between'
   },
   content: { flex: 1 },
+  noFlex: { flex: undefined },
```

Comment:
Not quite following why no flex is needed when there's a footer, but I trust.

Reply:
good question. Without this line, the footer gets stuck to the bottom of the screen even if there is only one or a few nft collections in the list. we want the footer to follow right after the list.